src/coreclr/vm/fcall.h

```diff
@@ -464,8 +221,6 @@ class CompletedFCallTransitionState
 
 #endif // !SWIZZLE_STKARG_ORDER
 
-#define HELPER_FRAME_DECL(x) HelperMethodFrame_##x##OBJ __helperframe
-
 // use the capture state machinery if the architecture has one
```

Comment:
The capture state machinery is gone. Delete everything from here till `FORLAZYMACHSTATE_ENDLOOP`